### PR TITLE
Check if streetAddress MUST NOT appear.

### PR DESCRIPTION
**Pull request by kumarde**

### CODE REVIEW

File: lints/lint_sub_cert_street_address_should_not_exist.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+type subCertStreetAddressShouldNotExist struct {
+	// Internal data here
+}
+
+func (l *subCertStreetAddressShouldNotExist) Initialize() error {
+	return nil
+}
+
+func (l *subCertStreetAddressShouldNotExist) CheckApplies(c *x509.Certificate) bool {
+	//Check if GivenName or Surname fields are filled out
```
Comment (zakird) [inline lints/lint_sub_cert_street_address_should_not_exist.go]: 🤕

File: lints/lint_sub_cert_street_address_should_not_exist.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	return util.IsSubscriberCert(c)
+}
+
+func (l *subCertStreetAddressShouldNotExist) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	//If all fields are absent
+	if len(c.Subject.GivenName) == 0 && len(c.Subject.Organization) == 0 && len(c.Subject.Surname) == 0 {
```
Comment (zakird) [inline lints/lint_sub_cert_street_address_should_not_exist.go]: Can you fix this ordering to humor me?

Comment (kumarde) [inline lints/lint_sub_cert_street_address_should_not_exist.go]: to what? Surname/Org/GivenName (that's the order of OID)

Comment (zakird) [inline lints/lint_sub_cert_street_address_should_not_exist.go]: I'd do the order in the BR. It's just awkward to have Org between two names.

Comment (kumarde) [inline lints/lint_sub_cert_street_address_should_not_exist.go]: okay, but i hate you

**Pull request merged**